inference/environments/minimal-ubuntu20.04-py38-cuda11.6.2-gpu-inference/asset.yaml

```diff
@@ -0,0 +1,10 @@
+name: minimal-ubuntu20.04-py38-cuda11.6.2-gpu-inference
```

Comment:
Is it critical that the full CUDA version be used (11.6.2 vs 11.6)? If you go this route you'd probably end up making a new environment for each patch version.

Reply:
The same name was registered in MCR so we are fine with this name.

Reply:
Yes, I see that. But I also see recent images that have been pushed to MCR via a process other than the registry-related ones. Only one process should be pushing to MCR for a particular image.

Reply:
Only one process should be pushing these images. We had a similar PR in AzuremlCli repo which as part of PR gates pushed the image here? We abandoned that PR to create new environments in this repo. Other than that no other process should be pushing these images.

Reply:
You'll need to find the other process that's pushing images to MCR if you want to reuse the existing MCR image name. We can't have two competing processing pushing to MCR. A new image was pushed there a week ago: https://mcr.microsoft.com/en-us/product/azureml/minimal-ubuntu20.04-py38-cuda11.6.2-gpu-inference/tags

Reply:
Arun just pointed out that the existing images I keep pointing to are missing the `/curated` part of the image name. No naming conflict after all--sorry!